sql/src/test/java/io/crate/analyze/SetAnalyzerTest.java

```diff
@@ -229,4 +229,29 @@ public void testResetLoggingSetting() {
         ResetAnalyzedStatement analysis = analyze("RESET GLOBAL \"logger.action\"");
         assertThat(analysis.settingsToRemove(), Matchers.<Set<String>>is(ImmutableSet.of("logger.action")));
     }
+
+    @Test
+    public void testSetLicense() throws Exception {
```

Comment:
Nitpick: Would be cool to also include some form of expected outcome in the test name. One good thing to ask yourself when writing the test-method-name is "Would I be able to re-create the test if I didn't know it's current implementation"